### PR TITLE
Jest fake-timer support

### PR DESCRIPTION
**Pull request by 2xic**
Thanks for creating this wonderful runner :) 

PR implements the base of jest fake-timer.

### CODE REVIEW

File: src/global-setup.js [hunk 1]
```diff
@@ -20,4 +29,20 @@ globalThis.jest = {
   clearAllMocks: jestMock.clearAllMocks.bind(jestMock),
   resetAllMocks: jestMock.resetAllMocks.bind(jestMock),
   restoreAllMocks: jestMock.restoreAllMocks.bind(jestMock),
+  useFakeTimers: () => {
+    jestTimer.useFakeTimers();
+    return globalThis.jest;
+  },
```
Comment (fisker) [inline src/global-setup.js:35]:
```suggestion
  useFakeTimers() {
    jestTimer.useFakeTimers();
    return this;
  },
```

Will this work?

Comment (2xic) [reply src/global-setup.js]: Yep, pushed a fix 👍️

**Review by nicolo-ribaudo — APPROVED** [review]
Sorry it took so long to land this, thank you! :)

File: src/global-setup.js [hunk 2]
```diff
@@ -20,4 +29,20 @@ globalThis.jest = {
   clearAllMocks: jestMock.clearAllMocks.bind(jestMock),
   resetAllMocks: jestMock.resetAllMocks.bind(jestMock),
   restoreAllMocks: jestMock.restoreAllMocks.bind(jestMock),
+  useFakeTimers() {
+    jestTimer.useFakeTimers();
+    return this;
```
[the remark this answers is not in the document]
Comment (fisker) [reply src/global-setup.js:34]: Isn't `this` in method of object equals the object?

[the remark this answers is not in the document]
Comment (nicolo-ribaudo) [reply src/global-setup.js:34]: Uh well, `this` is defined if you call `jest.useFakeTimers().something`. Does it work in Jest if you do `(jest.useFakeTimers, 0)().something`?

[the remark this answers is not in the document]
Comment (SimenB) [reply src/global-setup.js:34]: > Isn't `this` in method of object equals the object?

Hah, TIL 😀 

> Does it work in Jest if you do `(jest.useFakeTimers, 0)().something`?

`(jest.useFakeTimers , 0) is not a function` which seems fair?  I get the same doing `(document.createElement, 0)()` in a browser

[the remark this answers is not in the document]
Comment (fisker) [reply src/global-setup.js:34]: He mean `(0, jest.useFakeTimers)().something`.

The following won't work either

```js
const {useFakeTimers} = jest
useFakeTimers().something
```

```js
jest.useFakeTimers.call(null).something
```

```js
const foo = {useFakeTimers: jest.useFakeTimers}
foo.useFakeTimers().something
```

**Comment by nicolo-ribaudo** [comment]
Uhh I just noticed I never release this, I'm doing it now.

**Pull request merged**